examples/export_metrics/otlp-grpc.py

```diff
@@ -10,7 +14,15 @@
 init(
     exporter={
         "type": "otlp-proto-grpc",
+        "endpoint": "http://localhost:4317",  # You don't need to set this if you are using the default endpoint
```

Comment:
should this http endpoint be in the grpc example?

Reply:
it is my understanding that grpc still uses http for transport. but in any case, here i'm just copying the default parameters from [otel](https://github.com/open-telemetry/opentelemetry-python/blob/36aaa84d17284de6c873eadc0b955d97fe06b07b/exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py#L169C64-L169C65). should i add a comment about it somewhere?

Reply:
Nah just wasn’t sure if it was a copy pasta!